scipy/stats/__init__.py

```diff
@@ -488,6 +488,7 @@
 from ._page_trend_test import page_trend_test
 from ._mannwhitneyu import mannwhitneyu
 from ._fit import fit
+from ._covariance import Covariance, CovViaPrecision, CovViaPSD
```

Comment:
```suggestion
from ._covariance import Covariance, CovViaPrecision
```
Didn't mean to make that public.